### ISSUE TITLE
[Bug] Unable to restore Instagram

### ISSUE THREAD

**Issue by gabrc52** [NeoApplications/Neo-Backup#367]
**Description**
I'm getting an error message when trying to restore certain apps; I've reproduced this with Instagram

**Steps To Reproduce**

1. Backup Instagram
2. Uninstall Instagram
3. Restore Instagram
4. Get an error message:

```
Errors occurred
Instagram: a: Backup archive at content:/ com.android.externalstorage.documents /tree/primary%3ABackups/document /primary%3ABackups%2 Fcom.instagram .android%2F2021-06-08-1 5-56-32-user_0 %2Fdata.tar.gz.enc is missing
```

(The file does exist).

**Expected behavior**
Instagram restores fine

**System Information(please complete the following information):**

- Device: Google Pixel 4a 5G
- Android Version: 11
- ROM: Stock
- App's Version: 6.0.1

-----------------------------

**Additional Notes**

* If you're on the latest Test version(from the TG-Channel or Matrix-Room) please report there directly, not here.

* If you're on the latest github version:
  
  * Please read the [FAQ](https://github.com/machiav3lli/oandbackupx/blob/master/FAQ.md) first.
  * Please read all the open Issues. If one does match yours, leave a comment there if there's some extra information you can provide.

* If you're not on the latest github version. Update, then make sure that the bug is still there. If it's go further with reporting.

Thanks for reporting.

**Comment by machiav3lli** [comment]
You sure have put the right encryption password?

**Comment by gabrc52** [comment]
Yes. In fact, it doesn't prompt me for it, but I was able to restore another app.

**Comment by ivoanjo** [comment]
I'm having the exact same issue with Facebook messenger and a few other apps. I use LineageOS 18.1 (Android 11), and just did a clean reflash (trying to solve some unrelated issues) and then tried to restore most of my apps using oandbackupx.

Most apps worked fine (same encryption password for all of them), but indeed a few of them failed with this exact error. I've looked at the folders and the expected file seems to be there as well.

**Comment by machiav3lli** [comment]
Is there any logs of the issue? Otherwise, is there any libre apps (open-source and don't connect to proprietary networks) that I can test/check?

**Comment by ivoanjo** [comment]
Damn, I recently nuked it all and installed the apps by hand. I'll report if I can reproduce again 😭

**Comment by hg42** [comment]
should be fixed with #390

**Comment by hg42** [comment]
nothing happened...
